pom.xml

```diff
@@ -74,7 +74,7 @@
     <webservices.restVersion>2.29.0</webservices.restVersion>
     <patientsummaryVersion>2.2.0</patientsummaryVersion>
     <serializationVersion>0.2.15</serializationVersion>
-    <commonReportsVersion>1.5.0-SNAPSHOT</commonReportsVersion>
+    <commonReportsVersion>1.6.0-SNAPSHOT</commonReportsVersion>
```

Comment:
Why set this to SNAPSHOT and not keep it to release version? Are you expecting changes in the Common Reports in particular?